cmd/ddev/cmd/testdata/TestRealWorldDependencies/mock_redis/docker-compose.redis.yaml

```
@@ -0,0 +1,8 @@
+#ddev-generated
+# Mock Redis Docker Compose configuration
+services:
+  redis:
+    image: redis:alpine
+    container_name: ddev-${DDEV_SITENAME}-redis
+    ports:
```

It's mocking, so exposing ports not needed:

```suggestion
    expose:
```